### ISSUE TITLE
Elements in channel settings not keyboard accessible 

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1596]
**Priority** - medium, most users can still use the app as usual, this impacts only community owners that want to delete channel.
**Issue** - whole area is not focusable therefore it’s not keyboard accessible
**Solution** - all active elements should be keyboard accessible by being keyboard focusable. Don’t forget about making them descriptive. 
<img width="574" alt="Screenshot 2023-06-27 at 16 09 21" src="https://github.com/TryQuiet/quiet/assets/68909168/b4886b50-9a20-4a66-a7d7-6a87cf8256f4">
